.github/workflows/npm-release-publish.yml

```diff
@@ -71,6 +71,25 @@ jobs:
           git tag ${{ steps.version.outputs.version }}
```

Comment:
Do we still need this step or can it be removed?

Reply:
(oh we do if we use a different name to the version - see other comment)

Reply:
Not sure if the tag_name parameter in next step is to create a new tag or just reuse tags we have, since it is working well, will check this in, and let's test it when next release